my_internet_speed/__init__.py

```diff
@@ -31,7 +31,7 @@ def save(self):
         return Result.create(**data)
 
     def tweet(self):
-        if self.result.download >= MINIMUM_SPEED:
+        if not TWEET or self.result.download >= MINIMUM_SPEED:
```

Comment:
This should be `not TWITTER` instead of `not TWEET` (we are checking if is there a Twitter API set, not if is there a tweet text set).